docker-compose.yml

```diff
@@ -1,33 +1,14 @@
 version: "3"
 
 services:
-    app:
```

Comment:
For dev environment, we are using dockerized Postgres. We will be tackling fully dockerized dev env in #5.